x-pack/plugins/cases/public/components/connectors/jira/use_get_fields_by_issue_type.tsx

```diff
@@ -57,8 +57,8 @@ export const useGetFieldsByIssueType = ({
         });
 
         if (!didCancel.current) {
-          setIsLoading(false);
           setFields(res.data ?? {});
+          setIsLoading(false);
```

Comment:
I don't think these changes are necessary but I think we want to set loading to false only after we've set the data in state. Otherwise I think it's possible that a render could happen when seeing the `isLoading` set to false but maybe before the fields is set with data 🤷‍♂️